tests/Generating/BasicGeneratorTest.cpp

```diff
@@ -217,7 +218,94 @@ static void testGenerateNether()
 
 
 
+/** Storage for checksums with chunk coords. */
+struct CoordsWithChecksum
+{
+	cChunkCoords mCoords;
+	AString mChecksum;
+	CoordsWithChecksum(int aChunkX, int aChunkZ, const AString & aChecksum):
+		mCoords(aChunkX, aChunkZ),
+		mChecksum(aChecksum)
+	{}
+};
+
+
+
+
+
+/** Checks that the specified generator generates chunk that match the specified checksums. */
+static void checkChunkChecksums(
+	cChunkGenerator & aGenerator,
+	const std::vector<CoordsWithChecksum> & aCoordsWithChecksum,
+	const AString & aDimension
+)
+{
+	LOG("Testing the repeatability of the %s generator", aDimension.c_str());
```

Comment:
```suggestion
	LOG("Testing the repeatability of the %s generator", aDimension);
```

Reply:
I don't think a `std::string` is allowed to be passed through `...` varargs, I seem to remember this being an undefined behavior.

Reply:
It doesn't use c-style varargs any more
https://github.com/cuberite/cuberite/blob/d8d1e6e1d2f668cb7b34eae7484e59e2310e847a/src/LoggerSimple.h#L21

Reply:
Okay, template magic... :P